index.ts

```diff
@@ -1,17 +1,13 @@
-interface Document {
+export interface Document {
   declaration: {
-    attributes: {};
+    attributes: Record<string, any>;
```

Comment:
```suggestion
    attributes: Record<string, string>;
```

Should this be `string` instead of `any`?